src/data/currency_symbols.json

```diff
@@ -0,0 +1,176 @@
+{
+    "codes": [
+        "AED",
```

Comment:
Let's add POLY & POLYX to this file as well please

Reply:
It's already there